Package.swift

```diff
@@ -10,7 +10,7 @@ let package = Package(
     ],
     dependencies: [
       .package(url: "https://github.com/JohnSundell/Files.git", from: "4.1.1"),
-        .package(url: "https://github.com/kareman/Moderator.git", from: "0.5.1"),
+        .package(url: "https://github.com/DanielCech/Moderator.git", from: "0.5.1"),
```

Comment:
This change is necessary. Moderator is a common dependency with ScriptToolkit and using two different versions of the same library can cause problems.